### PR TITLE
[front] ManageSubscription: no access to stripe portal in trial

### PR DESCRIPTION
**Pull request by thervier**
## Description

Make sure ManageSubscriptionPage only tries to access the stripe portal when not in trial or without any stripeSubscriptionId (as already done in SubscriptionPage)

## Tests

None

## Risk

Low, corner case error management

## Deploy Plan

Deploy front

### CODE REVIEW

File: front/components/pages/workspace/subscription/ManageSubscriptionPage.tsx [hunk 1]
```diff
@@ -34,7 +42,7 @@ export function ManageSubscriptionPage() {
     }
 
     void redirectToStripePortal();
-  }, [owner.sId, router]);
+  }, [owner.sId, subscription, router]);
```
Comment (rfrenoy) [inline front/components/pages/workspace/subscription/ManageSubscriptionPage.tsx]: nit: no need to trigger the effect on subscription changes not affecting the guarding fields

```suggestion
  }, [owner.sId, subscription.trialing, subscription.stripeSubscriptionId, router]);
```

**Review by rfrenoy — APPROVED** [review]

File: front/components/pages/workspace/subscription/ManageSubscriptionPage.tsx (hunk 1 above)
Comment (thervier) [reply front/components/pages/workspace/subscription/ManageSubscriptionPage.tsx]: thx for the feedback, fixed

**Comment by thervier** [comment]
Forgot about this PR apparently... Too old now, closing it
